### DISCUSSION TITLE
[Crashlytics] recordError(error, jsErrorName) - jsErrorName not working on android in prod

### DISCUSSION THREAD

**Discussion by vitali-kastsiuk** [invertase/react-native-firebase#5114] [Q&A]
I use `recordError(error: Error, jsErrorName?: undefined | string): void;` with `jsErrorName` and it works on iOS for both dev and prod (i see separated Non-fatals), but on android I still see `.java <unknown>` output for all my logs(only for prod build, dev is OK). What could be wrong?

**Comment by andersonaddo** [comment]
its probably because prod builds are minimized and obfuscated.
you'll probably want to upload sourcemaps.

**Comment by viktorkurochka** [comment]
I used oficial documentation
https://firebase.google.com/docs/crashlytics/get-deobfuscated-reports?platform=android
It dosen't work for me(

**Reply by mikehardy** [reply]
I swear there was discussion (or an issue) on this just recently but I can no longer find it.
It seemed to be related to hermes and android minify being on (in any combination of the three sub-toggles that are available underneath the main "minifyEnabled" gradle setting). There has not been deeper analysis but something is making it so javascript stacks, when run in a minified android app, come out as useless.

No progress yet, sorry, but it's an area of investigation in the community at least

**Reply by julianD77** [reply]
For info - this appears to be the related ticket on this issue: https://github.com/invertase/react-native-firebase/issues/5642
